src/main/java/adris/altoclef/tasks/container/LootContainerTask.java

```diff
@@ -57,34 +61,49 @@ protected Task onTick(AltoClef mod) {
         }
         Optional<Slot> optimal = getAMatchingSlot(mod);
         if (optimal.isEmpty()) {
+            _weDoneHere = true;
             return null;
         }
-        setDebugState("Looting a container for all of their " + target.toString());
+        setDebugState("Looting items: " + targets);
         return new ClickSlotTask(optimal.get());
     }
 
     @Override
     protected void onStop(AltoClef mod, Task task) {
-        if (!_wasProtected) {
-            mod.getBehaviour().removeProtectedItems(target);
+        if (ContainerType.screenHandlerMatches(ContainerType.CHEST)) StorageHelper.closeScreen();
```

Comment:
Why check for chests? Why not just close screen all the time?

If you still wanna do this I'd check my later comment about making this consider all containers, not just chests.

Reply:
Oh I thought it might be weird if there wasn't even a screen open